R/qenv-within.R

```diff
@@ -1,34 +1,28 @@
-#' Evaluate expression in `qenv` object.
+#' Evaluate Expression in `qenv`
 #'
-#' Convenience function for evaluating inline code inside the environment of a `qenv`.
+#' @details
+#' `within` is a convenience function for evaluating inline code inside the environment of a `qenv`.
+#' It is a method for the `base` generic that wraps `eval_code` to provide a simplified way of passing code.
+#' `within` accepts only inline expressions (both simple and compound) and allows for injecting values into `expr`
+#' through the `...` argument:
+#' as `name:value` pairs are passed to `...`, `name` in `expr` will be replaced with `value`.
 #'
-#' This is a wrapper for `eval_code` that provides a simplified way of passing code for evaluation.
-#' It accepts only inline expressions (both simple and compound) and allows for injecting values into `expr`
-#' through the `...` argument: as `name:value` pairs are passed to `...`,
-#' `name` in `expr` will be replaced with `value`.
-#'
-#' @section Using language objects:
+#' @section Using language objects with `within`:
 #' Passing language objects to `expr` is generally not intended but can be achieved with `do.call`.
 #' Only single `expression`s will work and substitution is not available. See examples.
 #'
-#' @param data `qenv` object
-#' @param expr `expression` to evaluate
-#' @param ... `name:value` pairs to inject values into `expr`
+#' @param data (`qenv`)
+#' @param expr (`expression`) to evaluate. Must be inline code, see `Using language objects...`
```

Comment:
I think this param inherited from `base::within` looks also good.

Reply:
I want to refer to the section below, so it can't be inherited. But there is a `seealso base::within`.